src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +27,406 @@ export const AboutApp: React.FC = () => {
+        <Text variant="xl" mt={1}>
+          Here to Help You Find the Art You Love
+        </Text>
+        <Text variant="sm" mt={1}>
+          Artsy specialists are available to all Artsy members, wherever you are
+          in your collector’s journey. Our specialists join Artsy from leading
+          auction houses, galleries, and museums, and are ready to help you find
+          works that you love—with your price range, taste, and current
+          collection in mind.
+        </Text>
+        <Text variant="sm" mt={4}>
+          More questions about collecting?
+        </Text>
+        <Flex>
+          <Button
+            variant="primaryBlack"
+            width="40%"
+            // @ts-ignore
+            as={RouterLink}
+            to="/meet-the-specialists"
+            mt={4}
+            mr={1}
+          >
+            Contact Specialist
+          </Button>
+          <Button
+            variant="primaryWhite"
+            width="40%"
+            // @ts-ignore
+            as={RouterLink}
+            to="https://support.artsy.net/hc/en-us"
+            mt={4}
+            borderColor="black15"
+          >
+            Collector Resources
+          </Button>
+        </Flex>
+      </Column>
+      <Column span={6} mt={6}>
+        {AdvisorySpecialists.map(specialist => (
+          <Box width="100%">
+            <Flex flexDirection="row">
+              <Avatar size="md" src={specialist.photo.url} mr={2} />
+              <Flex flexDirection="column">
+                <Text variant="lg">{specialist.name}</Text>
+                <Text variant="md">{specialist.title}</Text>
+                <Text variant="md" color="black60" mb={2}>
+                  {specialist.location}
                 </Text>
-              </Box>
-              <Box>
-                <Image
-                  width="320px"
-                  height="480px"
-                  src="https://artsy-media-uploads.s3.amazonaws.com/KeKyoFW83QKNeST23YyQIg/about_Rebecca.jpg"
-                />
-                <Text>REBECCA BRONFEIN RAPHAEL</Text>
-              </Box>
-              <Box py={20}>
-                <Box py={20}>
-                  <Text>More questions about collecting?</Text>
-                </Box>
-                <GridColumns>
-                  <Column span={6}>
-                    <Button>Collector Resources</Button>
-                  </Column>
-                  <Column span={6}>
-                    <Button>Contact Specialist</Button>
-                  </Column>
-                </GridColumns>
-              </Box>
-            </Column>
-          </GridColumns>
-        </Box>
-      </Box>
-    </>
+                <RouterLink to={`mailto:${specialist.email}`}>
+                  {specialist.email}
+                </RouterLink>
+              </Flex>
+            </Flex>
+            <Spacer my={2} />
+            <Text borderBottom="1px solid" borderBottomColor="black15" />
+            <Spacer my={2} />
```

Comment:
This last spacer would count as an external margin — golden rule of margins = always use internal margins never outer margins.